### ISSUE TITLE
Bix Dev PedcbioPortal Support: load GTEx RNA-Seq to PedCBio

### ISSUE THREAD

**Issue by runjin326** [PediatricOpenTargets/ticket-tracker#275]
#### [Required] How would you most generally describe your issue? 
- [x] Update data for project in pedcbio portal
- [ ] Questions related to data on pedcbio portal 

Within V10 release, there are GTEx RNA-Seq samples that can be loaded to PedCBio. 
This is mentioned in a previous [ticket ](https://github.com/PediatricOpenTargets/ticket-tracker/issues/229) but want to separate this one out for clarity.

**Comment by migbro** [comment]
Just saw this - partly blocked by PR https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/163.
Also created an "internal" ticket that is related here: https://github.com/d3b-center/bixu-tracker/issues/1323

**Comment by jharenza** [comment]
After looking at the [QA load ](https://kf-strides-cbioportal-qa.kidsfirstdrc.org/study/summary?id=ped_opentargets_2021) and discussions with @runjin326 and @migbro, we need to do some additional tweaking of case ID lists here before this gets released to prod. @runjin326 will be creating a master pedcbio load issue to encompass all issues we will want to address for the v10 load, with an estimated delivery of end april

**Comment by jharenza** [comment]
Update case lists to say "All Samples" instead of "All tumors" since GTEX is now included.

**Comment by jharenza** [comment]
closing this in favor of #294
